frontend/src/components/Admin/DevPortfolio/DevPortfolioDetails.tsx

```diff
@@ -29,6 +30,20 @@ const DevPortfolioDetails: React.FC<Props> = ({ uuid, isAdminView }) => {
       <Header textAlign="center" as="h3">
         Deadline: {new Date(portfolio.deadline).toDateString()}
       </Header>
+      <Button
+        onClick={() => {
+          DevPortfolioAPI.regradeSubmissions(portfolio.uuid)
+            .then((portfolio) => setPortfolio(portfolio))
+            .catch((e) =>
+              Emitters.generalError.emit({
+                headerMsg: 'Failed to regrade all submissions',
+                contentMsg: 'Please try again or contact the IDOL team'
+              })
+            );
+        }}
+      >
+        Regrade Latest Submissions
```

Comment:
I think this should say just "Regrade Submissions" because all submissions are going to be rerun, not just the latest ones.